Assignment 2.Rmd

```diff
@@ -80,15 +92,77 @@ plot(g,layout=layout.fruchterman.reingold, vertex.color=VERTEX$gender)
 #We can change the thickness of the edge according to the number of times a particular teacher has sent a particular student to the principal.
 
 plot(g,layout=layout.fruchterman.reingold, vertex.color=VERTEX$gender, edge.width=EDGE$count)
-````
+```
 
 ## Part II
-
 In Part II your task is to [look up](http://igraph.org/r/) in the igraph documentation and create a graph that sizes the student vertices in terms of the number of disciplinary actions they have recieved, and the teachers in terms of the number of disciplinary actions they have given out.
 
+```{r}
+VERTEX$group<-as.factor(VERTEX$group)
+plot(g,layout=layout.auto,vertex.size=EDGE$count*3,vertex.color=VERTEX$group)
+```
```

Comment:
Looks good! You are almost there, you need to make sure that each student and teacher is only referred to once in the Vertex list.